specification/containerservice/DeploymentSafeguards.Management/deploymentsafeguards.tsp

```diff
@@ -0,0 +1,91 @@
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using Azure.Core;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Foundations;
+using TypeSpec.OpenAPI;
+using Azure.Core.Traits;
+
+@armProviderNamespace
+namespace Microsoft.ContainerService;
+
+@doc("Deployment Safeguards")
+@resource("deploymentSafeguards")
+@singleton("default")
+model DeploymentSafeguard
+  is Azure.ResourceManager.ExtensionResource<DeploymentSafeguardsProperties> {
+  @key("deploymentSafeguardName")
+  @doc("Required default name")
+  @segment("deploymentSafeguards")
+  @visibility(Lifecycle.Read)
+  @pattern("default")
+  @path
+  name: string;
+
+  ...EntityTagProperty;
+}
+
+@doc("Specifies the level of Deployment Safeguards")
+union DeploymentSafeguardsLevel {
+  @doc("Best practice violations will only show warnings")
+  Warn: "Warn",
+
+  @doc("Best practice violations will be denied")
+  Enforce: "Enforce",
+
+  string,
+}
+
+@doc("DeploymentSafeguards Properties")
+model DeploymentSafeguardsProperties {
+  @doc("Provisioning State")
+  @visibility(Lifecycle.Read)
+  provisioningState?: ProvisioningState;
+
+  @doc("The deployment safeguards level. Possible values are Warn and Enforce")
+  level: DeploymentSafeguardsLevel;
+
+  @doc("User defined list of namespaces to exclude from Deployment Safeguards. Deployments in these namespaces will not be checked against any safeguards")
+  excludedNamespaces?: Array<string>;
+
+  @doc("System defined list of namespaces excluded from Deployment Safeguards. These are determined by the underlying provider (such as AKS), and cannot be changed. Deployments in these namespaces will not be checked")
+  @visibility(Lifecycle.Read)
+  systemExcludedNamespaces: Array<string>;
```

Comment:
I'd like to confirm if this property `systemExcludedNamespaces` is a really required one for this model. 

Considering it is readOnly(@visibility(Lifecycle.Read)) so it is absent in request but a required property in response, is my understanding correct?